File: mysql/jruby/update/mysql_update.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	mysql_update.rb
#
#				Jul/09/2011
#
include Java
import org.gjt.mm.mysql.Driver
import java.sql.DriverManager
import java.lang.System
#
require 'date'
#
load '/var/www/data_base/common/jruby_common/jruby_sql_manipulate.rb'
#
# ------------------------------------------------------------
puts "*** 開始 ***"
#
#
id_in = ARGV[0]
population_in = ARGV[1].to_i
#
puts id_in,population_in
#
str_connect = "jdbc:mysql://host_mysql/city"
#
conn = DriverManager.getConnection(str_connect,"scott", "tiger")

sql_update_proc(conn,id_in,population_in)
sql_read_proc(conn)

conn.close()
#
puts "*** 終了 ***"
#
# ------------------------------------------------------------

File: mysql/lua/update/mysql_update.lua
-- ----------------------------------------------------
--	mysql_update.lua
--
--					Jul/16/2011
-- ----------------------------------------------------
require	("luasql.mysql")
require	("text_manipulate")
require	("sql_manipulate")
-- ----------------------------------------------------
print ("*** 開始 ***")

local id_in=arg[1]
local population_in=0 + arg[2]

print (id_in,population_in)

local dbname = "city"
local dbhost = "host_mysql"
local dbuser = "scott"
local dbpass = "tiger"

local env = assert(luasql.mysql())
local con = assert(env:connect(dbname, dbuser, dbpass, dbhost))


sql_update_proc	(con,id_in,population_in)

con:close()
env:close()

print ("*** 終了 ***")
-- ----------------------------------------------------
